docs/site/downloads/cldr-47.md

```diff
@@ -75,7 +77,22 @@ For a full listing, see [¤¤BCP47 Delta](https://unicode.org/cldr/charts/dev/de
 
 ### [Locale Changes](https://unicode.org/cldr/charts/dev/delta/index.html)
 
-- TBD
+- Cleanups for current pattern variants `alt="alphaNextToNumber"` and `alt="noCurrency"`: These were introduced in CLDR 42
+(per [CLDR-14336](https://unicode-org.atlassian.net/browse/CLDR-14336)) to provide a cleaner way of adjusting currency
+patterns when an alphabetic currency symbol is used, or when a currency-style pattern is desired without a currency symbol
+(as for use in a table). Some further adjustments were needed ([CLDR-17879](https://unicode-org.atlassian.net/browse/CLDR-17879)):
+    - Adjust coverage so that these variants are at moderate (not comprehensive) coverage for standard/accounting currency formats with
+    `numberSystem="latn"`, and so that `alt="alphaNextToNumber"` is at modern (not comprehensive) for oither relevant number systems in
+    in a locale. Coverage was already correct for other combinations of these attributes with various numberSystems.
+    - Adjust PathHeader so compact currency for relevant non-Latn number systems in a locale will appear in Survey Tool.
+    - In root, add an `alt="alphaNextToNumber"` variant for the standard/accounting currency patterns.
+    - Ensure that in the most commonly-used locales. for all relevant number systems in the locale, the standard/accounting currency
+    patterns have both `alt="alphaNextToNumber"` and `alt="noCurrency"` variants (inherting as necessary), and the compact currency
+    formats have the `alt="alphaNextToNumber"` variants.
```

Comment:
I think we need to state issues in terms on impact on clients of CLDR, not the nitty-gritty internals.
```suggestion
(as for use in a table). Gaps in the data coverage showed up, because the translators weren't shown the right values. 
Fixes were made in  [CLDR-17879](https://unicode-org.atlassian.net/browse/CLDR-17879).
```

Reply:
I could make this change if you want.

Reply:
Actually, I'm going to go ahead and approve this PR, we can just refine later.

Reply:
Thanks I agree. I will merge this now, we can create another PR for further changes.